src/github.com/getlantern/flashlight/genconfig/cloud.yaml.tmpl

```diff
@@ -1,6 +1,7 @@
 # cloud.yaml contains the default configuration that's made available on the
 # internet.
 client:
+  firetweetversion: "0.0.6"
```

Comment:
Actually while just doing a fresh Lantern build I feel like regenerating the config every time automatically is probably a good idea, as the certs, IPs, etc, could theoretically change over time, and it's worthwhile to periodically update it.